internal/file/file.go

```diff
@@ -32,80 +33,76 @@ Get retrieves a file from these locations (in order):
 
 - AWS S3
 
-If a file is found, then it is saved to disk and the path is returned. The caller is responsible for removing files when they are no longer needed.
+If a file is found, then it is saved as a temporary local file and the name is returned. The caller is responsible for removing files when they are no longer needed; files should be removed even if an error occurs.
 */
-func Get(ctx context.Context, path string) (string, error) {
-	file, err := os.CreateTemp("", "tempfile")
+func Get(ctx context.Context, location string) (string, error) {
```

Comment:
What do you think about adding `appconfig://` or similar scheme handling here that calls out to `appconfig.GetPrefetch`? This could simplify the fetching of the [AWS app's config](https://github.com/brexhq/substation/pull/38/files#diff-3bcf387acd4e8e562b8e7f5d9934d6e7205f0cc26e4047feb65a86d0ec42243bR69-R76) a bit.